unity/CAVE/Assets/Resources/CLI.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.IO;
+using System.Xml;
+using System.Text.RegularExpressions;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+using UnityEditor.SceneTemplate;
+
+public class CLI : MonoBehaviour
+{
+    static void Main()
+    {
+        string xmlPath = null;
+
+        // Get command line arguments from Python
+        try {
+            string[] args = System.Environment.GetCommandLineArgs();
+            for(int i = 0; i < args.Length; i++)
+            {
+                if(args[i] == "--xmlPath") xmlPath = args[++i];
```

Comment:
Is this incrementing `i`? Maybe just do `i + 1`. And you can `break` if it found `xmlPath`

Reply:
Will do! I may have to pass in more variables but I'll handle that refactor if I get there